sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1254,63 +1284,92 @@ static void *_gnrc_netif_thread(void *args)
     gnrc_netif_release(netif);
 
     while (1) {
+        int msg_waiting = 0;
+#ifdef MODULE_GNRC_NETIF_ASYNC_EVENTS
+        /* Using messages for ordered, synchronous IPC, and events for
```

Comment:
`GNRC_NETAPI_MSG_TYPE_SND` isn't synchronous, and also events aren't any less ordered or unordered than `msg_t` so I don't really get this statement...